Agenda-client/src/components/NewHelpRequest/NewHelpRequest.test.tsx

```diff
@@ -0,0 +1,53 @@
+import { render, screen, fireEvent } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+import NewHelpRequest from './NewHelpRequest';
+import { Widget, WidgetType } from '../widget/Widget';
+
+import { jest } from '@jest/globals';
+
+jest.mock('../../utils/MyCrypto', () => {
+  let id = 0;
+  return {
+    randomUUID: jest.fn().mockReturnValue('' + id++),
+  };
+});
+
+class TestWidget extends Widget {
+  constructor(type: WidgetType, dataId?: string) {
+    super(type, dataId);
+  }
+}
+// beforeAll(() => {
```

Comment:
is this needed?

Reply:
I just pushed again. the comment has already been deleted.